### PR TITLE
Update Supervisor Documentation and Tutorials for 0.56.0

### PR DESCRIPTION
**Pull request by cnunciato**
This change updates the Supervisor-related tutorials on habitat.sh to bring them in line with the changes coming in the next release.

Signed-off-by: Christian Nunciato <chris@nunciato.org>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

**Comment by cnunciato** [comment]
Putting a do-not-merge on this until the release goes out.

File: www/source/demo/process-supervisor/steps/10.html.slim [hunk 1]
```diff
@@ -35,11 +35,15 @@ classes: demo_step
         = code(:shell) do
           | $ hab studio enter
         = code(:shell) do
-          | [1][default:/src:0]# hab start yourorigin/sample-node-app --strategy at-once
+          |
+            [1][default:/src:0]# hab sup run
```
Comment (christophermaier) [inline www/source/demo/process-supervisor/steps/10.html.slim]: If this is being done in a studio (line 36), then there's already a supervisor running in the background.

**Comment by christophermaier** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/385230593).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/385230593).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Comment by christophermaier** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/385374048).

**Comment by thesentinels** [comment]
:broken_heart: Travis CI reports [this PR failed to deploy](https://travis-ci.org/habitat-sh/habitat/builds/385374048).

The next step is to examine the [job](https://travis-ci.org/habitat-sh/habitat/builds/385374048) and figure out why. If it is transient, you can try re-triggering the Travis CI Job - if it passes, this PR will be automatically merged. If it is not transient, you should fix the issue and update this pull request, and issue `approve` again. If you believe it will never pass, and you are feeling :godmode:, you can issue a `force` to merge this PR anyway.

**Comment by christophermaier** [comment]
@thesentinels acceptance

**Comment by thesentinels** [comment]
:metal: I am sending your branch against master to our acceptance environment.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/386306025).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/386306025).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**